markwiki/application.py

```diff
@@ -7,6 +7,7 @@
 
 from markwiki.search.engine import SearchEngine
 from markwiki.storage.factory import UserStorageFactory
+from markwiki.git.git_integration import GitIntegration
```

Comment:
I like to see imports in alphabetic order so this line should move up a couple of spots.